aspnet/publishing/iis.rst

```diff
@@ -291,6 +312,17 @@ Troubleshooting
 
 - Confirm that the Application Pool is not in the `Stopped` state.
 
+IISIntegration module was not injected
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Browser:** 502.5 Process Failure
+- **Application Log:** Process was created with commandline '"dotnet" ...' but either crashed or did not reponse within given time or did not listen on the given port '2140', ErrorCode = '0x800705b4'
+- **ASP.NET Core Module Log:** Log file created and shows normal operation.
+
```

Comment:
Ok ... got it. I'd recommend providing the full browser error page title. List both of the Application Log exceptions by app type and pull the port number from each (I've used "PORT" elsewhere here). I'd suggest firming up "normal operation" as "that the application started normally," just so that they know exactly what the log is going to say. I'd suggest ...

**Browser:** HTTP Error 502.5 - Process Failure
**Application Log (Portable app):** Process was created with commandline '"dotnet" ...' but either crashed or did not reponse within given time or did not listen on the given port 'PORT', ErrorCode = '0x800705b4'
**Application Log (Self-contained app):** Process was created with commandline '"PATH\my_application.exe" ' but either crashed or did not reponse within given time or did not listen on the given port 'PORT', ErrorCode = '0x800705b4'
**ASP.NET Core Module Log:** Log file created and shows that the application started normally.